libs/libvtrutil/src/vtr_bimap.h

```diff
@@ -31,6 +31,8 @@ namespace vtr {
 template<class K, class V, template<typename...> class Map = std::map, template<typename...> class InvMap = std::map>
```

Comment:
@vaughnbetz and @hzeller  - This commit adds the copy, move and initializer_list constructors for the bimap data structure in the libvtrutill library. I needed to use bimap with the initializer list constructor in the Odin-II codebase, and I found that it has no constructor definition. Previously, the bimap data structure did not support the initializer list constructor and followed C++ default behaviours for copy and move constructors. I could only add the initializer_list constructor to this library. However, I found the copy and move constructors' definition beneficial if we suppose to have a custom initializer list constructor. It would be appreciated if you could let me know your thoughts on these changes.